src/docs/implementations/statsD.adoc

```diff
@@ -1,4 +1,4 @@
-= Micrometer Influx
```

Comment:
Weird. I wonder why [this page](https://micrometer.io/docs/registry/statsD) doesn't show "Influx" at the top.

Reply:
@shakuzen FYI when I was working on https://github.com/micrometer-metrics/micrometer-docs/pull/85, I noticed this value isn't used for the title. We can remove them all if there's no benefit having it.